src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14.06px;
+  text-align: left;
+
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.logo {
+  align-content: center;
+  height: 40px;
+  width: 40px;
+  text-decoration: none;
+}
+
+.nav__item {
```

Comment:
Be consistent with your vertical margins. It's better to choose either top or bottom margin for elements like '.nav__item' to avoid collapsing margins.